lib/ansible/modules/find.py

```diff
@@ -372,6 +378,10 @@ def statinfo(st):
     }
 
 
+def handle_walk_errors(e):
```

Comment:
I think it'd be clearer what it does if you name it like this
```suggestion
def raise_walk_error(e):
```
(it'd be more obvious when used in the code below)

Reply:
it really does both `handle_walk_error_by_reraising'  was a bit too long